src/components/SpruceForm/FieldTemplates.tsx

```diff
@@ -150,7 +150,7 @@ const ArrayItem: React.VFC<
   return useExpandableCard ? (
```

Comment:
this comment placement is random. 

Can you please remove [this](https://github.com/evergreen-ci/spruce/blob/main/src/pages/projectSettings/tabs/NotificationsTab/SubscriptionField.tsx#L8-L9)? It's no longer needed.